docs/source/using_gridappsd/Query Timeseries.rst

```diff
@@ -0,0 +1,55 @@
+Query Time-series database
+-------------------------
+
+Connect to the running GridAPPS-D container
+
+.. code-block:: bash
+
+  user@foo>docker exec -it gridappsddocker_gridappsd_1 bash
+
+..
+
+Now we are inside the executing container. Start the platform.
+
+.. code-block:: bash
+
+  root@737c30c82df7:/gridappsd# ./run-docker.sh
+
+..
+
+Open your browser to http://localhost:8080/ and click the menu button.
```

Comment:
Are you sure this is how you want this section to look?
![image](https://user-images.githubusercontent.com/3979063/40862729-4256faf2-65a2-11e8-91af-6577c5dd064c.png)